cumulusci/robotframework/tests/test_template_util.py

```diff
@@ -1,9 +1,7 @@
-import unittest
-
 from cumulusci.core import template_utils
 
 
-class TemplateUtils(unittest.TestCase):
+class TemplateUtils:
```

Comment:
This doesn't have the right naming convention to be detected by pytest as a test class